signer/local_cosigner.go

```diff
@@ -250,6 +306,7 @@ func (cosigner *LocalCosigner) sign(req CosignerSignRequest) (CosignerSignRespon
 	return res, nil
 }
 
+// Implements ThresholdEd25519Signature interface
 func (cosigner *LocalCosigner) dealShares(req CosignerGetEphemeralSecretPartRequest) (HrsMetadata, error) {
```

Comment:
These should move out to a new file for the "local" threshold ed25519 signature implementation.